### ISSUE TITLE
FQDN pool member missing its FQDN information

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-telemetry-streaming#171]
### Environment
 * Telemetry Streaming Version: 1.23
 * BIG-IP Version: 14.1.4

### Summary
We use the Datadog consumer (I believe it's not a consumer-specific issue).

Our pool members are FQDN nodes.
In the consumer trace, we see such metrics for the pool members:

```
                    {
                        "host": "ip-10-100-2-55.eu-west-1.compute.internal",
                        "interval": 60,
                        "metric": "f5_test_another.pools.members.serverside.bitsOut",
                        "points": [
                            [
                                1632493905.325,
                                0
                            ]
                        ],
                        "tags": [
                            "addr:::",
                            "monitorStatus:fqdn-up",
                            "port:443",
                            "availabilityState:available",
                            "enabledState:enabled",
                            "status.statusReason:The DNS server(s) are available"
                        ],
                        "type": "gauge"
                    },
```

We miss the `FQDN` information (as a tag), because we might have multiple entries with the same addr / port

![image](https://user-images.githubusercontent.com/17331976/135489451-93d7f180-8a4a-42fc-8c65-0ce5ddfd4bbb.png)

Please add the FQDN tag to all pool members metrics

**Comment by dstokesf5** [comment]
Here is a sample from the iControl REST output for an FQDN pool member.
```json
{
    "kind": "tm:ltm:pool:members:membersstate",
    "name": "_auto_142.250.189.164:443",
    "partition": "Common",
    "fullPath": "/Common/_auto_142.250.189.164:443",
    "generation": 25308,
    "selfLink": "https://localhost/mgmt/tm/ltm/pool/test/members/~Common~_auto_142.250.189.164:443?ver=16.1.0",
    "address": "142.250.189.164",
    "connectionLimit": 0,
    "dynamicRatio": 1,
    "ephemeral": "true",
    "fqdn": {
        "autopopulate": "enabled",
        "tmName": "www.google.com"
    },
    "inheritProfile": "enabled",
    "logging": "disabled",
    "monitor": "default",
    "priorityGroup": 0,
    "rateLimit": "disabled",
    "ratio": 1,
    "session": "user-enabled",
    "state": "unchecked"
}
```
The auto-tagging only looks at sibling properties to the metric. In this case `fqdn` is an object and not a string, so it gets skipped. This is currently working as designed. I am migrating this to a feature request to look into additional auto-tagging options that would pick up fqdn data. Thank you for your feedback.

**Comment by amolari** [comment]
I guess it has been fixed in v1.26.0?

**Comment by dstokesf5** [comment]
This has been resolved in the 1.26.0 release.
